code/modules/vending/_vending.dm

```diff
@@ -753,8 +783,8 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 		get_asset_datum(/datum/asset/spritesheet_batched/vending),
 	)
 
-/obj/machinery/vending/ui_state(mob/user)
-	return GLOB.default_state
+//obj/machinery/vending/ui_state(mob/user)
```

Comment:
I couldn't figure out. What does this actually do?

I uncommented it and couldn't see any difference. Is it used as a reference for some other thing in TGUI?

Reply:
This does nothing, default state is the default UI state anyway. You can remove this code